packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1323,6 +1323,7 @@ export class ProjectService {
         metricQuery,
         queryTags,
         invalidateCache,
+        warehouse,
```

Comment:
this naming is really confusing because the function is called `getResultsFromCacheOrWarehouse` but `warehouse` has nothing to do with the `warehouse` in the name.